coffee-warehouse/src/components/GridComponents/IsOnlineComponent.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <span

```

Comment:
since the element is the same - 'span'  I would rather just change the content and the class depending on the 'is_online' instead of changing the element

Reply:
Fixed with commit https://github.com/telerik/kendo-vue/pull/164/commits/e751c2b1201e03800b1523454456af731cbfa252